app/form_objects/support/privileged_user_form.rb

```diff
@@ -0,0 +1,45 @@
+class Support::PrivilegedUserForm
+  include ActiveModel::Model
+
+  attr_accessor :full_name, :email_address, :privileges
+
+  validates :full_name, presence: { message: 'Enter the name of the user' }
+  validates :email_address, presence: { message: 'Enter the email address of the user' },
+                            format: { with: /\A.*@computacenter.com|.*@digital.education.gov.uk|.*@education.gov.uk\z/,
+                                      message: 'Email address must be end with @computacenter.com, @digital.education.gov.uk or @education.gov.uk' }
+  validate :validate_email_not_taken
+  validate :validate_atleast_one_privilege
```

Comment:
I'd probably add an underscore so the method becomes `validate_at_least_one_privilege`